src/lsg/patterns/element/index.tsx

```diff
@@ -45,69 +45,116 @@ export interface StyledPlaceholder {
 }
 
 const StyledElement = styled.div`
-	cursor: default;
 	position: relative;
+	z-index: 1;
 `;
 
 const StyledElementLabel = styled.div`
```

Comment:
@LKuechler Is it possible to apply styling on the element during dragging around? So like opacity or scale?

Reply:
Solved with 71c298d81a3a4fa0c4f046c6a159f704028ce5b9